README.md

```diff
@@ -1,8 +1,46 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Kevin Subhash . Jarvis Consulting
 
+I am a recent graduate from University of Toronto with an Honours Bachelor of Science. I did a double major in Computer Science and Mathematics during my undergrad. Over the years, I have been exposed to many concepts such as web development, data structures/algorithms, software design, and systems programming just to name a few. I first started programming in grade 10 and found it very fascinating which has led me to pursue a career in computer science. I am currently a software developer at Jarvis where I was introduced to many new technologies such as Docker, Maven, and Springboot. That being said, I see myself as an inspired individual who is committed to professional growth and continuous learning.
```

Comment:
"...at Jarvis where I was introduced to..." - you have been working with these technologies. You have been creating applications using Java, Maven, Docker, Springboot, and others. "Introduced to" does not suggest you know anything about significant about this tech stack. Similar for "...been exposed to...": try for something more active to demonstrate your experience.

Remove "That being said". It unnecessarily weakens your final statement. Just say you are an inspired individual...